pages.es/common/more.md

```diff
@@ -0,0 +1,19 @@
+# more
+
+> Abre un archivo para lectura interactiva, permitiendo desplazar y buscar (Solamente en una dirección, adelante).
```

Comment:
1. `more` accepts scroll backwards in macOS and Linux now.
2. `more` is an old page. You can add the more info link to the English page too.
3. I think `navegar` sounds better.
```suggestion
> Abre un archivo para lectura interactiva, permitiendo navegar y buscar.
> Más información: <https://manned.org/more>.
```